api/src/middlewares/services/EmailService/NotifyService/personalisation/personalisationBuilder.userConfirmation.ts

```diff
@@ -16,6 +16,7 @@ export function buildUserConfirmationPersonalisation(answers: AnswersHashMap, me
   const docsList = buildUserConfirmationDocsList(answers, isSuccessfulPayment);
   const country = answers["country"] as string;
   const post = answers["post"] as string;
+  const multipleNameChanges = answers["nameChanged"] === "name changed more than once";
```

Comment:
can we change the value to be something more code friendly? "changedMoreThanOnce" or something?